docs/content/endpoints/download.md

```diff
@@ -1 +1,8 @@
 # /download
+
+To download a song, all you need is the Beatmap ID.
+
+## /download/key/{Beatmap Key}
```

Comment:
> This applies to all added headings

Instead of `{Beatmap Key}` please use the standard path format. eg: `/download/key/:key`